webservices/src/main/java/com/javaschool/webservices/model/PackageType.java

```diff
@@ -0,0 +1,37 @@
+package com.javaschool.webservices.model;
+
```

Comment:
tip: Take a look to lombok library to avoid boilerplate code.
[https://projectlombok.org/](url)

Reply:
lomnok implemented avoiding bolerplate code.